src/index.ts

```diff
@@ -150,4 +203,29 @@ function compareOperations(a: CodegenOperation, b: CodegenOperation): number {
 	return 0
 }
 
+/**
+ * Loop over an array of CodegenOperation and return true if it contains an operation with a multipart/form-data request.
+ * @param operations
+ */
+function containsMultipartOperation(operations: CodegenOperation[]): boolean {
+	for (const operation of operations) {
+		if (operation.requestBody?.defaultContent.mediaType.mimeType === 'multipart/form-data') { 
+			return true
+		}
+	}
+	return false
+}
+
+/**
+ * An interface representing file properties in a multipart/form-data request.
+ */
+type FileProperty = {
```

Comment:
This should be `interface FileProperty {` instead... usually I'd put interface definitions at the top of the file.